config.js

```diff
@@ -5,6 +5,10 @@ var config = {
     //
 
     hosts: {
+
+        // parent document jitsi path
+        parent: 'https://superhero.com/meet',
```

Comment:
What happens when we have superhero, stage, dev, prod? Do they still use same parent host?

Reply:
It's different instances that use different config. @venimus can confirm that.

Reply:
@Chima1707 not sure how to track that in repo.

Reply:
@venimus Please can you suggest something here?

Reply:
it is ok like this for now, will make changes per sever

Reply:
Ok, great!